src/parser/semantic_check.c

```diff
@@ -2532,6 +2533,8 @@ pt_get_compatible_info_from_node (const PT_NODE * att, SEMAN_COMPATIBLE_INFO * c
   cinfo->prec = cinfo->scale = 0;
   cinfo->ref_att = att;
 
+  /* TODO - cinfo->force_cast should also be inited */
```

Comment:
`cinfo->force_cast` is not initialized here. 
Please also confirm other routines. I don't see any but want to have your observations.

Reply:
I am well aware of this problem and I have put a comment in the pull req: